### PR TITLE
updated docs to explain the new default threshold setting for ArynPartitioner

### PR DESCRIPTION
**Pull request by dtecuci**

### CODE REVIEW

File: docs/source/aryn_cloud/aps_output.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Takes the format `[x1, y1, x2, y2]` where each coordinate is given as the propor
       "page_number": 1-indexed page number the element occurs on }
 ```
 
-The `score` is the number used to determine whether to keep the element based on the `threshold`. Elements with a confidence score higher than the threshold will be kept as the model is "confident" in its prediction for this element whereas elements with a confidence score lower than this threshold will be removed, as the model is less confident in its prediction for this element.
+The `score` is the model's "confidence" in its prediction for that particular bounding box. By dafault the model makes its best prediction, but the user can control this using the `threshold` parameter (defaults to "auto"). If the user specifies a numeric value between 0 and 1, only Elements with a confidence score higher than the specified threshold value will be kept.
```
Comment (sohamkasar19) [inline docs/source/aryn_cloud/aps_output.md:81]: Misspelled default here.

Comment (dtecuci) [reply docs/source/aryn_cloud/aps_output.md:81]: I wouldn't. We're keeping that post-processing private for now.

File: docs/source/sycamore/transforms/partition.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Parameters:
 
 * ```use_partitioning_service```: If ```True```, the partitioner uses the *Aryn  Partitioning Service*. Defaults to ```True```.
 * ```model_name_or_path```: The HuggingFace coordinates or model local path. It defaults to ```SYCAMORE_DETR_MODEL```, and you should only change it if you are testing a custom model. Ignored when ```use_partitioning_service``` is ```True```.
-* ```threshold```: The threshold to use for accepting the models predicted bounding boxes. A lower value will include more objects, but may have overlaps; a higher value will reduce the number of overlaps, but may miss legitimate objects. It defaults to ```0.4```.
+* ```threshold```: This represents the threshold for accepting the model's predicted bounding boxes. It defaults to "auto", where the model make its best prediction. The user can specify a numerical threshold between 0 and 1. A lower value will include more objects, but may have overlaps, a higher value will reduce the number of overlaps, but may miss legitimate objects. We suggest trying values around ``0.32``.
```
Comment (bsowell) [inline docs/source/sycamore/transforms/partition.md:24]: Maybe change the recommendation at the end to use auto? You can maybe say something like "We recommend using the default "auto" threshold. If you do set the threshold manually, we recommend starting with a value of 0.32."

**Review by bsowell — APPROVED** [review]

File: docs/source/aryn_cloud/aps_output.md (hunk 1 above)
Comment (sohamkasar19) [reply docs/source/aryn_cloud/aps_output.md:81]: Cool

File: docs/source/sycamore/transforms/partition.md (hunk 2 above)
Comment (dtecuci) [reply docs/source/sycamore/transforms/partition.md]: Updated. thanks!

**Comment by dtecuci** [comment]
merged

File: docs/source/sycamore/transforms/partition.md (hunk 2 above)
Comment (jonfritz) [reply docs/source/sycamore/transforms/partition.md]: Is the model making the best prediction, or is it using the post processing logic?

**Pull request merged**